README.md

```diff
@@ -90,8 +111,12 @@ Define a function called `doSomething`. It should take one argument, called
 `thingToDo`. When called, `doSomething` should invoke the `thingToDo` function. Demonstrate calling `doSomething` while passing in an argument.
 
 Your Answer:
-```js
-// write code here
+```
+function doSomething(thingToDo) {
+  var xyz = thingToDo;
```

Comment:
Yes, very close and this question can be tricky. The only thing missing is the first example we are expecting thingToDo to be invoked, 
`thingToDo();`